longest_palindromic_substring/longest_palindromic_substring.cpp

```diff
@@ -0,0 +1,52 @@
+/*
+*  Implementation of dp problem: Longest Palindromic substring
+*  Time complexity: O(n^2)
+*  Space complexity: O(n^2)
+*  n is size of input string
+*/
+
+#include <iostream>
+#include <string>
+#include <vector>
+
+using namespace std;
+
+string longest_palindromic_substring(string test) {
```

Comment:
add a multi line comment describing the function and it;s parameters and what is returns